js/src/modal.js

```diff
@@ -410,10 +410,23 @@ class Modal {
         return
       }
 
+      const isModalOverflowing = this._element.scrollHeight > document.documentElement.clientHeight
+
+      if (!isModalOverflowing) {
+        this._element.style.overflowY = 'hidden'
+      }
+
       this._element.classList.add(CLASS_NAME_STATIC)
-      const modalTransitionDuration = getTransitionDurationFromElement(this._element)
+      const modalTransitionDuration = getTransitionDurationFromElement(this._dialog)
+      EventHandler.off(this._element, TRANSITION_END)
```

Comment:
what's the point of adding `EventHandler.off(this._element, TRANSITION_END)` when we use `once` just after that?

Reply:
Sorry for very late reply! This `EventHandler.off(this._element, TRANSITION_END)` is for addressing quick clicks (click backdrop again while the transition is not finished). If this line is missing, the scroll bar will be displayed.